level1/f256/modules/vtio.asm

```diff
@@ -74,9 +75,28 @@ AltISR
 * Handle keyboard (if available)
                     ldx       V.KeyDrvEPtr,u
                     cmpx      #$0000
-                    beq       HandleSound
-                    jsr       6,x call AltIRQ routine in keydrv
-                    
+		    ifgt      Level-1
+                    beq       HandleMSTimer
+		    else
+		    beq	      HandleSound
+		    endc
+                    jsr       6,x			call AltIRQ routine in keydrv
+		    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU.  Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.